--- gnome-robots-50.0/src/meson.build.orig	2026-03-18 22:29:23.000000000 +0100
+++ gnome-robots-50.0/src/meson.build	2026-10-04 20:04:53.017136287 +0200
@@ -11,6 +11,7 @@ run_command(
 )
 
 cargo_options = [ '--manifest-path', meson.project_source_root() / 'Cargo.toml' ]
+cargo_options += [ '--target', 'x86_64-unknown-linux-gnux32' ]
 cargo_options += [ '--target-dir', meson.project_build_root() / 'src' ]
 
 if profile == 'debug'
@@ -38,7 +39,7 @@ robots = custom_target(
         cargo_env,
         cargo, 'build', cargo_options,
         '&&',
-        'cp', 'src' / rust_target / meson.project_name(), '@OUTPUT@',
+        'cp', 'src' / 'x86_64-unknown-linux-gnux32' / rust_target / meson.project_name(), '@OUTPUT@',
     ]
 )
 
